### ISSUE TITLE
Inconsistent Examples

### ISSUE THREAD

**Issue by JAK1047** [MicrosoftDocs/office-docs-powershell#7318]
The examples have the operator as -OperationId. The Parameters has it as -Identity. Using the latest pre-release of the module seems to have neither as accepted.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 44ec2119-d178-b9f8-3199-481ecd84eea3
* Version Independent ID: 6e814ba4-7680-12b5-10ee-f7c71865cd71
* Content: [Get-CsBatchPolicyAssignmentOperation (MicrosoftTeamsPowerShell)](https://docs.microsoft.com/en-us/powershell/module/teams/get-csbatchpolicyassignmentoperation?view=teams-ps)
* Content Source: [teams/teams-ps/teams/Get-CsBatchPolicyAssignmentOperation.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/teams/teams-ps/teams/Get-CsBatchPolicyAssignmentOperation.md)
* Service: **teams-powershell**
* GitHub Login: @tomkau
* Microsoft Alias: **tomkau**

**Comment by get-itips** [comment]
Hello @JAK1047 how are you? Hope you are doing ok

_OperationId_ is an **alias** of _Identity_, that is detailed in the parameter for -Identity

I could verify what you say that in 1.1.11-preview both parameters have been removed, however, the documentation will reflect the behavior of the General Availability module, once this change is merged into the GA version we will update the article.
Thank you!

**Comment by JAK1047** [comment]
Ok so the documentation is more for what it will reflect in final release rather than the current state?

**Comment by get-itips** [comment]
> Ok so the documentation is more for what it will reflect in final release rather than the current state?

The GA module still uses Identity (and OperationId), the General availability module is the current production-ready module of Teams PowerShell, for more information on the difference see here https://docs.microsoft.com/en-us/microsoftteams/teams-powershell-overview please.
